inc/saiswitch.h

```diff
@@ -634,6 +634,22 @@ typedef enum _sai_packet_trim_queue_resolution_mode_t
 
 } sai_packet_trim_queue_resolution_mode_t;
 
+/**
+ * @brief Global PTP configuration
+ */
+typedef enum _sai_switch_ptp_mode_t
+{
+    /** None - per-port configuration can be used to switch PTP on for a port */
+    SAI_SWITCH_PTP_MODE_NONE,
+
+    /** One-step - all ports use one-step PTP mode */
+    SAI_SWITCH_PTP_MODE_SINGLE_STEP_TIMESTAMP,
+
+    /** Two-step - all ports use two-step PTP mode */
```

Comment:
Same as above. We can still have one or more ports with  SAI_PORT_ATTR_PTP_MODE == SAI_PORT_PTP_MODE_ONE_STEP_TIMESTAMP. These ports will NOT use two-step PTP mode.